src/MCPServer/lib/linkTaskManagerReplacementDicFromChoice.py

```diff
@@ -46,27 +46,33 @@ def __init__(self, jobChainLink, pk, unit):
         for i, dic in enumerate(dicts):
             self.choices.append((i, dic.description, dic.replacementdic))
```

Comment:
FYI: self.choices is the list that fills up the drop-down in the user interface when the workflow needs intervention from the user.